1E datadog integration changes (#1707)
DataDog/integrations-extras: 1e/README.md

Our platform contains 1E for Visibility, Control, and Compliance, 1E for Digital Experience Observability, and 1E for Service Desk Automation solutions.

The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account.

Integrating Datadog with 1E enables IT administrators to:

- View device data and related information in one place in Datadog
- View Digital Employee Experience (DEX) score and related information
- Build a customizable dashboard inside Datadog for customer specific insights.

## Setup

### Prerequisites

To configure your integration between Datadog and 1E you will need to have the following in place:

- 1E 8.4
  - platform installed, with at least one 1E client connected to provide metrics data
  - a user that is assigned to the **Full Administrator** role
  - 1E PowerShell Toolkit 2.0 or later, available from the [1E Support portal][2]
- Datadog
  - a user that is assigned to the **Datadog Admin** role and able to download the 1E Dashboard from the Marketplace.

### Configuration

Steps required to configure your integration between Datadog and 1E.

#### Creating a Datadog API Key

> To set up the integration with 1E, you need to upload a Datadog API key and site URL to your 1E
> installation, using the 1E PowerShell Toolkit or Postman.

Reviewer: ```suggestion
To set up the integration with 1E, create a Datadog API key, and then upload the API key and site URL to 1E using the 1E API.```

> 1. To get started, go to the [Datadog APIs page][3], where you can create an API key
> 
> 2. Determine your [Datadog API site URL][4] with reference to Datadog documentation

Reviewer: ```suggestion
1. Navigate to the **Organization Settings** > [**API Keys**][3] page and create an API key.
2. Identify your [Datadog site URL][4]. You will need this information when adding the Datadog configuration to 1E.
```

# 1E

## Overview

> At [1E][1], we reimagine how technology serves people and create new ways for IT to shape the future of work.

Reviewer 2: Could you provide 1-2 sentences on 1E does? We try to avoid marketing copy in the overview.

Author: Added sentence on what do we do at 1E

Integrating Datadog with 1E enables IT administrators to:

- View device data and related information in one place in Datadog
- View Digital Employee Experience (DEX) score and related information
- Build a customizable dashboard inside Datadog for customer specific insights.

## Setup

### Prerequisites

To configure your integration between Datadog and 1E you will need to have the following in place:

- 1E 8.4
  - platform installed, with at least one 1E client connected to provide metrics data
  - a user that is assigned to the **Full Administrator** role
  - 1E PowerShell Toolkit 2.0 or later, available from the [1E Support portal][2]
- Datadog
  - a user that is assigned to the **Datadog Admin** role and able to download the 1E Dashboard from the Marketplace.

### Configuration

Steps required to configure your integration between Datadog and 1E.

#### Installing the 1E Dashboard

You can install the 1E Dashboard from the Datadog Marketplace at any time in this process.

To install, navigate to **Integrations** in your Datadog instance, search for **1E**, and open **1E Dashboard**.

In 1E Dashboard, click on **Install Integration**.

> The dashboard's tiles will show data after you have configured integration with 1E.

Reviewer 2: Is there a separate integration for the dashboard? If not, the customer will simply need to: 
1. Go to the Datadog integrations page
2. Search for 1E 
3. Install the integration

This will automatically populate the dashboard

Author: Removed this part

#### Creating a Datadog API Key

To set up the integration with 1E, you need to upload a Datadog API key and site URL to your 1E
installation, using the 1E PowerShell Toolkit or Postman.

1. To get started, go to the [Datadog APIs page][3], where you can create an API key

> 2. Determine your [Datadog API site URL][4] with reference to Datadog documentation

Reviewer 2: Customers can create an API key directly from the web app. Do they need to use postman?

Author: To upload Datadog API key and site URL, 1E PowerShell Toolkit should be used and recommended approach.

However, user can use postman or other tools to post API request to 1E platform

If you are familiar with using 1E, then you can confirm data is available by signing in to the 1E portal and viewing in the **Experience Analytics** app. You need to sign in as a user that is assigned in 1E to either the **Full Administrator** role or **Experience Viewers** role.

Please refer to [Datadog documentation][5] for more details about creating, adding, and customizing your dashboards.

### Using 1E Dashboard

You can use the 1E Dashboard to view digital employee experience scores, trends and device metrics reported by 1E in Datadog. Experience data helps you visualize your end-users' experience of IT service delivery across your enterprise.

Once you have the 1E Dashboard, you can view 1E metrics like:

- Application crashes
- Application hangs
- Service failures
- OS reboots
- OS Upgrades
- Software installations and uninstalls
- Patch installations and uninstalls.

In addition, you can view data about:

- Number of connected devices
- Digital Employee Experience (DEX) score related to:
- Performance
- Stability
- Responsiveness
- Sentiment.

### Schedule reports and create notifications in Datadog

> Once you have configured the 1E Dashboard, you have the option to schedule reports and create
> notifications from it.
> Refer to the [Datadog documentation][5] for details.

Reviewer 2: ```suggestion
Once you have configured the 1E Dashboard, you have the option to schedule reports and create
notifications from it. Refer to the [Datadog documentation][5] for details.
```

## Data Collected

### Metrics

See [metadata.csv][3] for a list of metrics provided by this integration.

### Service Checks

The 1E integration does not include any service checks.

### Events

The 1E integration does not include any events.

## Troubleshooting

> Need help? Contact [Datadog support][6] or reach out to [1E Support][1].

Reviewer 2: ```suggestion
Need help? Contact [1E Support][1].
```
